docs/modules/databases/neo4j.md

```diff
@@ -21,6 +21,14 @@ You are not limited to Unit tests and can of course use an instance of the Neo4j
 
 ## Additional features
 
+### Custom password
+
+A custom password can be provided:
+
+<!--codeinclude-->
+[Disable authentication](../../../modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java) inside_block:withAdminPassword
```

Comment:
```suggestion
[Custom password](../../../modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java) inside_block:withAdminPassword
```

Reply:
Thanks...and done. Seems like I need to improve my copy 'n paste skills 😄